remote_server_ip:
    ip0: "192.168.20.245"     
    ip1: "192.168.20.24"
    ip2: "192.168.20.102"
    ip3: "192.168.20.103"
    ip4: "192.168.20.104"
    ip5: "192.168.20.105"
    ip6: "192.168.20.106"
    ip7: "192.168.20.107"
    ip8: "192.168.20.108"
    ip9: "192.168.20.109"

server_port:
    port0:  9000
    port1:  9001
    port2:  9002
    port3:  9003
    port4:  9004
    port5:  9005
    port6:  9006
    port7:  9007
    port8:  9008
    port9:  9009

common:
    cluster_num: 2           #include self,max=10
    self_id: 0